With change to integration_test.go only, I got the following error (reproducing the failure we've seen when trying to use v5.0.0)
```
  Thu Jul  7 16:03:35 UTC 2022: \033[96msrc/busybench.ts\033[0m:\033[93m14\033[0m:\033[93m27\033[0m - \033[91merror\033[0m\033[90m TS2307: \033[0mCannot find module '@google-cloud/profiler' or its corresponding type declarations.
            Thu Jul  7 16:03:35 UTC 2022: 
            Thu Jul  7 16:03:35 UTC 2022: \033[7m14\033[0m import * as profiler from '@google-cloud/profiler';
            Thu Jul  7 16:03:35 UTC 2022: \033[7m  \033[0m \033[91m                          ~~~~~~~~~~~~~~~~~~~~~~~~\033[0m
            Thu Jul  7 16:03:35 UTC 2022: 
            Thu Jul  7 16:03:35 UTC 2022: \033[96msrc/busybench.ts\033[0m:\033[93m33\033[0m:\033[93m5\033[0m - \033[91merror\033[0m\033[90m TS2304: \033[0mCannot find name 'setTimeout'.
            Thu Jul  7 16:03:35 UTC 2022: 
```

If I can make this test pass, that's a a sign that the build process has been fixed.

--- a/package.json
+++ b/package.json
@@ -70,7 +70,6 @@
   "files": [
     "build/src",
     "build/third_party/cloud-debug-nodejs",
-    "bindings",
     "build/protos"
   ],
   "nyc": {

--- a/system-test/integration_test.go
+++ b/system-test/integration_test.go
@@ -99,6 +99,7 @@ retry npm_install --nodedir="$NODEDIR"
 npm run compile
 npm pack --nodedir="$NODEDIR" >/dev/null
 VERSION=$(node -e "console.log(require('./package.json').version);")
+PROFILER="$HOME/cloud-profiler-nodejs/google-cloud-profiler-$VERSION.tgz"
 
 TESTDIR="$HOME/test"
 mkdir -p "$TESTDIR"
@@ -106,7 +107,7 @@ cp -r "system-test/busybench" "$TESTDIR"
 cd "$TESTDIR/busybench"
 
 retry npm_install @mapbox/node-pre-gyp --save
-retry npm_install --nodedir="$NODEDIR" typescript gts; npm link ../../cloud-profiler-nodejs
+retry npm_install --nodedir="$NODEDIR" "$PROFILER" typescript gts
 
 npm run compile
 

--- a/test/test-init-config.ts
+++ b/test/test-init-config.ts
@@ -20,7 +20,7 @@ import * as sinon from 'sinon';
 
 import {createProfiler, nodeVersionOkay} from '../src/index';
 import {Profiler} from '../src/profiler';
-import * as packageJson from '../package.json';
+const packageJson = require('../../package.json');
 
 describe('nodeVersionOkay', () => {
   const version = parseInt(packageJson.engines.node.replace('>=', ''));

